### ISSUE TITLE
[Go] Abc.com giving Adobe Pass error

### ISSUE THREAD

**Issue by sftman18** [ytdl-org/youtube-dl#25197]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.05.08**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
$ /usr/local/bin/youtube-dl -v https://abc.com/shows/general-hospital/episode-guide/2020-04/30-general-hospital-43020
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'https://abc.com/shows/general-hospital/episode-guide/2020-04/30-general-hospital-43020']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2020.05.08
[debug] Python version 2.7.18rc1 (CPython) - Linux-5.4.0-29-generic-x86_64-with-Ubuntu-20.04-focal
[debug] exe versions: ffmpeg 4.2.2, ffprobe 4.2.2, rtmpdump 2.4
[debug] Proxy map: {}
[Go] 30-general-hospital-43020: Downloading webpage
[Go] VDKA0_uixgx8ru: Downloading JSON metadata
ERROR: This video is only available for users of participating TV providers. Use --ap-mso to specify Adobe Pass Multiple-system operator Identifier and --ap-username and --ap-password or --netrc to provide account credentials.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 797, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 530, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/go.py", line 187, in _real_extract
    url, video_id, requestor_id, resource)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/adobepass.py", line 1414, in _extract_mvpd_auth
    raise_mvpd_required()
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/adobepass.py", line 1379, in raise_mvpd_required
    'and --ap-username and --ap-password or --netrc to provide account credentials.', expected=True)
ExtractorError: This video is only available for users of participating TV providers. Use --ap-mso to specify Adobe Pass Multiple-system operator Identifier and --ap-username and --ap-password or --netrc to provide account credentials.
```

## Description

I regularly download General Hospital from Abc.com.  This worked fine on 5/6/20, but failed yesterday.  I updated to the 5/8/20 release of youtube-dl, but that didn't fix the problem.
Thanks!

**Comment by sftman18** [comment]
I can confirm that #25216 fixes this problem.
Thanks!

**Comment by Azmaeus** [comment]
I verified the same. Using #25216 I'm able to download shows from abc again.

**Comment by magnum32** [comment]
> I verified the same. Using #25216 I'm able to download shows from abc again.

I use YouTube-DL Gui, so how will i be able to apply the fix?

**Comment by magnum32** [comment]
Its impossible for me to find the directory for the extractor on my computer. how can i locate at?

**Comment by Azmaeus** [comment]
> I use YouTube-DL Gui, so how will i be able to apply the fix?

Sorry @magnum32 , I don't have any experience with Youtube-DL Gui. I couldn't figure out how to extract and re-compile the Windows executable, I ended up just downloading the source code, making the change myself, and then running the python file to verify the fix.

**Comment by magnum32** [comment]
> > I use YouTube-DL Gui, so how will i be able to apply the fix?
> 
> Sorry @magnum32 , I don't have any experience with Youtube-DL Gui. I couldn't figure out how to extract and re-compile the Windows executable, I ended up just downloading the source code, making the change myself, and then running the python file to verify the fix.

is there a way that you could post the py files?

**Comment by magnum32** [comment]
I wonder when this fix will be implemented for people who use Youtube-dl GUI?

**Comment by sftman18** [comment]
Closing issue, as the fix in #25216 works for me on the side.
Thanks!

**Comment by Azmaeus** [comment]
> > > I use YouTube-DL Gui, so how will i be able to apply the fix?
> > 
> > Sorry @magnum32 , I don't have any experience with Youtube-DL Gui. I couldn't figure out how to extract and re-compile the Windows executable, I ended up just downloading the source code, making the change myself, and then running the python file to verify the fix.
> 
> is there a way that you could post the py files and your youtube-dl config?

You can go to https://github.com/tmthywynn8/youtube-dl/tree/Fix-issue-%2325207 and click "Clone or Download", then select the "Download Zip" option. This will download all the files you need. Inside the zip will be a "youtube-dl-Fix-issue-25207" folder, and within that folder a "youtube-dl" folder. There is where all the .py files are, you'll need to extract them somewhere. I don't know if or how to use those with YoutTube-DL Gui. To run it manually you would need to ensure you have Python installed, and your path (environment variables) set-up to allow python to work in the command prompt, then you could open a command prompt to that "youtube-dl" folder, then run "python __main__.py <web page of video>" (that's main surrounded by four underscores, two on each side; replace <web page of video>" including the ankle-braces with your link, e.g. https://abc.com/shows/bless-this-mess/episode-guide/season-02/20-tornado-season-part-two ).

**Comment by magnum32** [comment]
> > > > I use YouTube-DL Gui, so how will i be able to apply the fix?
> > > 
> > > 
> > > Sorry @magnum32 , I don't have any experience with Youtube-DL Gui. I couldn't figure out how to extract and re-compile the Windows executable, I ended up just downloading the source code, making the change myself, and then running the python file to verify the fix.
> > 
> > 
> > is there a way that you could post the py files and your youtube-dl config?
> 
> You can go to https://github.com/tmthywynn8/youtube-dl/tree/Fix-issue-%2325207 and click "Clone or Download", then select the "Download Zip" option. This will download all the files you need. Inside the zip will be a "youtube-dl-Fix-issue-25207" folder, and within that folder a "youtube-dl" folder. There is where all the .py files are, you'll need to extract them somewhere. I don't know if or how to use those with YoutTube-DL Gui. To run it manually you would need to ensure you have Python installed, and your path (environment variables) set-up to allow python to work in the command prompt, then you could open a command prompt to that "youtube-dl" folder, then run "python **main**.py " (that's main surrounded by four underscores, two on each side; replace " including the ankle-braces with your link, e.g. https://abc.com/shows/bless-this-mess/episode-guide/season-02/20-tornado-season-part-two ).

Than you!!!

**Comment by 2ShedsJackson** [comment]
> Another oddity is that this same "The View" link is now downloading an episode of the Conners. 

The link is no longer valid. You're downloading whatever video is showcased on the abc.com homepage.
